qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -555,7 +555,7 @@ def _wait_for_completion(
             except UserTimeoutExceededError:
                 raise IBMQJobTimeoutError(
                     'Timeout while waiting for job {}'.format(self._job_id))
-        self._update_status_position(ApiJobStatus(status_response['status']),
+        self._update_status_position(ApiJobStatus(status_response['status'].upper()),
```

Comment:
These two are actually unnecessary since `StatusResponseSchema` already mandates `status` value must be one of the `ApiJobStatus` values.